packages/std/src/addresses.rs

```diff
@@ -53,6 +59,12 @@ impl Deref for HumanAddr {
     }
 }
 
+impl PartialEq<str> for HumanAddr {
+    fn eq(&self, other: &str) -> bool {
+        self.0 == other
+    }
+}
```

Comment:
This is required when you want to do `assert_eq!(&human_addr, "cos934gh9034hg04g0h134");`. Otherwise `&HumanAddr` and `&str` cannot be compared directly.

Reply:
Awesome find!